src/core/linalg/src/sparse/4C_linalg_view.hpp

```diff
@@ -0,0 +1,102 @@
+// This file is part of 4C multiphysics licensed under the
+// GNU Lesser General Public License v3.0 or later.
+//
+// See the LICENSE.md file in the top-level for license information.
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#ifndef FOUR_C_LINALG_VIEW_HPP
+#define FOUR_C_LINALG_VIEW_HPP
+
+
+#include "4C_config.hpp"
+
+#include <boost/proto/detail/template_arity.hpp>
+
+#include <memory>
+
+FOUR_C_NAMESPACE_OPEN
+
+namespace Core::LinAlg
+{
+  /**
+   * A helper struct to easily specify that one of our linear algebra classes is a wrapper for
+   * another class. This struct should be specialized for each of our linear algebra classes and
+   * contain a single typedef `type` for the class that is wrapped.
+   */
+  template <typename T>
+  struct WrapperFor
+  {
+    static_assert("You need to specialize this struct for the wrapper class.");
+  };
```

Comment:
We specialize this struct for our wrappers as you can see for Vector and MultiVector in this PR.